rules-reviewed/quarkus1/quarkus1-10/quarkus1-11-mongodb-panache.windup.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="quarkus1-11-mongodb-panache"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints for upgrading to quarkus 1.11 for applications that use MongoDB Panache
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <!-- The major version is implied by the target, the minor is reflected in the verionRange -->
+        <sourceTechnology id="quarkus1" versionRange="(,10]"/>
+        <targetTechnology id="quarkus1" versionRange="[11,)"/>
```

Comment:
Is the intention of this ruleset to cover migration **from** any Quarkus version **older than 11** to any version **newer or equal to 11**, so this rule will be executed for any target 11, 12, 13, 14, etc... ?

Reply:
That's correct so you could be migrating from say 1.8 to 1.13. so you would want the rules that cover the 1.10  to 1.11 to fire.